hhs_oauth_server/settings/base.py

```diff
@@ -379,6 +408,10 @@
             'handlers': ['console'],
             'level': 'INFO',
         },
+        'audit.global_state_metrics': {
+            'handlers': ['console', 'kinesis'],
```

Comment:
Nice 👍 , this looks like the ideal way to handle this

Reply:
This doesn't need to change but just a thought: how are the logs getting to splunk? Is it basically collecting after it goes to the console?

Reply:
@nbragdon That's right. Splunk has a log ingestion agent that runs on the instances. It gets the logging directly from the log files.

Reply:
Just going to add in here for reference: The handlers don't seem to be async and if there is a long timeout or some other issue with the kinesis stream it'll hold up processing user requests. So we are looking at other options, either cloudwatch logs or an agent that pulls from the log files instead.

Reply:
+1 for this. The log handlers are blocking (not async), so just going to use this method related to the "audit.global_state_metrics" type logging for now. Those run on a nightly schedule via Jenkins & Django management command---so will not impact the web services, if they have issues.  Going to refactor that logging with this in mind next...